packages/babel-traverse/src/hub.js

```diff
@@ -1,6 +1,7 @@
-export default class Hub {
-  constructor(file, options) {
-    this.file = file;
-    this.options = options;
-  }
+export type Hub = {
+  mark: (type: string, message: string) => void;
+  addHelper: (name: string) => Object;
+  getScope: () => Object;
+  getCode: () => string;
+  buildError:(node: Object, msg: string, Error: Error) => Error;
```

Comment:
I don't know if some users rely on this class.

I would like to keep the class there, what do you think? I'm not sure because the class is empty anyway.